README.md

```diff
@@ -1,11 +1,11 @@
 # MongooseIM platform
 
-[![Build Status](https://github.com/esl/MongooseIM/workflows/CI/badge.svg)](https://github.com/esl/MongooseIM/actions)
-[![CircleCI](https://circleci.com/gh/esl/MongooseIM.svg?style=shield&circle-token=53cdda11cfb95bd7a7044c509c89cb73d3fdeb5a)](https://github.com/esl/MongooseIM/)
-[![codecov](https://codecov.io/gh/esl/MongooseIM/branch/master/graph/badge.svg)](https://codecov.io/gh/esl/MongooseIM)
+[![GitHub Actions](https://github.com/esl/MongooseIM/workflows/CI/badge.svg?branch=master)](https://github.com/esl/MongooseIM/actions?query=workflow%3ACI+branch%3Amaster)
+[![CircleCI](https://circleci.com/gh/esl/MongooseIM.svg?style=shield&circle-token=53cdda11cfb95bd7a7044c509c89cb73d3fdeb5a)](https://app.circleci.com/pipelines/github/esl/MongooseIM)
```

Comment:
If the link is not publicly available, do leave the link to the source repo instead. Don't put links nobody can access.

Reply:
Likewise for the index.md file and the other circleci link below.

Reply:
If you check commit verification statuses they all have the same links to CircleCI, the same story with our PRs.
so these links are everywhere in our repo, and being honest I find it quite confusing when the link that reports verification status lead to the repo itself. I would rather want to see some message from CircleCI that I have no access to the results. At least I would know that it's CircleCI.

Reply:
It's difficult to decide for me, I'm fine with either version as long as it is consistent.

Reply:
Ok, not insisting then here either 👍🏽